miio/waterheater.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from typing import Any, Dict
+
+from .click_common import command, format_output
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WaterHeaterStatus:
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    @property
+    def washStatus(self) -> str:
+        return self.data["washStatus"]
+
+    @property
+    def velocity(self) -> str:
+        return self.data["velocity"]
+
+    @property
+    def waterTemp(self) -> str:
```

Comment:
```suggestion
    def water_temperature(self) -> str:
```
Is this a string or integer or float or something else?